google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -122,6 +122,7 @@ public void run() {
     commandLine.withWorkDirectory(stagingDirectory);
     consoleLogLn(loggingHandler, "Working directory set to: " + stagingDirectory.getAbsolutePath());
     commandLine.withParentEnvironmentType(ParentEnvironmentType.CONSOLE);
+    commandLine.getEnvironment().put("CLOUDSDK_METRICS_ENVIRONMENT", "gcloud-intellij");
```

Comment:
@etanshaul 

This is something we should be setting for all gcloud command invocations.  Can we factor out gcloud initialization for setting this and the user creds?